maint.tf

```
@@ -0,0 +1,77 @@
+module "state_bucket" {
+  source        = "git::https://github.com/plus3it/terraform-aws-tardigrade-s3-bucket.git?ref=5.0.0"
+  bucket        = var.backend_config.bucket
+  force_destroy = var.backend_config.force_destroy
+  versioning    = var.backend_config.versioning
+
+  public_access_block = {
+    block_public_acls       = var.backend_config.public_access_block.block_public_acls
+    block_public_policy     = var.backend_config.public_access_block.block_public_policy
+    ignore_public_acls      = var.backend_config.public_access_block.ignore_public_acls
+    restrict_public_buckets = var.backend_config.public_access_block.restrict_public_buckets
+  }
```

Pretty sure this can be simplified:

```suggestion
  public_access_block = var.backend_config.public_access_block
```